partials/mixins/panel.html

```diff
@@ -0,0 +1,5 @@
+<div id="{{toggle}}-panel" class="reveal js-hidden">
+    <div class="panel-indent{{#getErrorClass}}{{toggle}}{{/getErrorClass}}">
```

Comment:
There are situations where we have text before the field (and validation message) and we don't want the border infringing on this.